src/questions/general-questions.md

```diff
@@ -4,32 +4,33 @@ layout: layouts/page.njk
-* Describe how you would create a simple slideshow page.
-* If you could master one technology this year, what would it be?
-* Explain the importance of standards and standards bodies.
-* What is Flash of Unstyled Content? How do you avoid FOUC?
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
-* Explain some of the pros and cons for CSS animations versus JavaScript animations.
-* What does CORS stand for and what issue does it address?
-* How did you handle a disagreement with your boss or your collaborator?
-* What resources do you use to learn about the latest in front end development and design?
-* What skills are needed to be a good front-end developer?
-* What role do you see yourself in?
-* Explain the difference between cookies, session storage, and local storage?
+- What did you learn yesterday/this week?
+- What excites or interests you about coding?
+- What is a recent technical challenge you experienced and how did you solve it?
+- Can you explain what happens when you enter a URL into a browser?
+- When building a new web site or maintaining one, can you explain some techniques you have used to increase performance?
+- Can you describe some SEO best practices or techniques you have used lately?
+- Can you explain any common techniques or recent issues solved in regards to front-end security?
+- What actions have you personally taken on recent projects to increase maintainability of your code?
+- Talk about your preferred development environment.
+- Which version control systems are you familiar with?
+- Can you describe your workflow when you create a web page?
+- If you have 5 different stylesheets, how would you best integrate them into the site?
+- Can you describe the difference between progressive enhancement and graceful degradation?
+- How would you optimize a website's assets/resources?
+- How many resources will a browser download from a given domain at a time?
+  - What are the exceptions?
+- Name 3 ways to decrease page load (perceived or actual load time).
+- If you jumped on a project and they used tabs and you used spaces, what would you do?
+- Describe how you would create a simple slideshow page.
+- If you could master one technology this year, what would it be?
+- Explain the importance of standards and standards bodies.
+- What is Flash of Unstyled Content? How do you avoid FOUC?
+- Explain what ARIA and screenreaders are, and how to make a website accessible.
+- Explain some of the pros and cons for CSS animations versus JavaScript animations.
+- What does CORS stand for and what issue does it address?
+- How did you handle a disagreement with your boss or your collaborator?
+- What resources do you use to learn about the latest in front end development and design?
+- What skills are needed to be a good front-end developer?
+- What role do you see yourself in?
+- Explain the difference between cookies, session storage, and local storage?
```

Comment:
Can you minimize the number of lines changed to JUST the new question.

Reply:
done ! @roblarsen